src/Lykke.Service.Assets/Controllers/v2/AssetPairsController.cs

```diff
@@ -13,20 +15,25 @@ namespace Lykke.Service.Assets.Controllers.V2
     [Route("api/v2/asset-pairs")]
     public class AssetPairsController : Controller
     {
-        private readonly IAssetPairService _assetPairService;
+        private readonly IAssetPairService  _assetPairService;
+        private readonly ICache<IAssetPair> _cache;
```

Comment:
Better to put the cache management into the new Managare layer, or at least to the Service, but not controller. Api v1 should be with cache to